presto-docs/src/main/sphinx/functions/array.rst

```diff
@@ -29,9 +29,9 @@ Array Functions
 
```

Comment:
I also updated the documentation to make the variable names consistent, sometimes the array variable is ``array`` and most of the time it was ``x``. I made them all ``x``.